pkg/daemon/daemon.go

```diff
@@ -124,13 +140,21 @@ func New(
 		}
 		glog.Infof("Booted osImageURL: %s (%s)", osImageURL, osVersion)
 	}
+
+	currentBootIDBytes, err := ioutil.ReadFile("/proc/sys/kernel/random/boot_id")
```

Comment:
In the future I'd like us to try to move things like this into functions. Not needed now though.

Reply:
Factored out into a new function locally.